lesson-2/web-store/src/pages/admin_page.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
```

Comment:
the file should be named with `hyphen` instead of `underscore`

the file name should refer component it's exporting

admin-page.js -> class AdminPage